nulib/meadow
# Bump max_concurrency on multipart upload to 50

# Summary 

The `CopyFileToPreservation` step is taking very long for very large files.
We want to try to up the concurrency of the multipart upload to S3 and see if it speeds up. 
If it does we want to go as high as possible w/out causing problems.


# Specific Changes in this PR
- bump `max_concurrency` on multipart upload to 50
- make concurrency configurable per environment


# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

- Probably not possible to test in dev environment. Just make sure existing functionality works
- On staging/prod, send through an ingest sheet with 4 250GB files. See if copy file to pres takes less than an hour and a half +

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
